### DISCUSSION TITLE
How to change/setup network mode in 802.15.4

### DISCUSSION THREAD

**Discussion by marijnacht** [zephyrproject-rtos/zephyr#45305] [Q&A]
Hi, I'm investigating the use of Zephyr for our current subGHz network products. We currently use the TI CC1352 with their proprietary stack and like to switch to IP based networking. I have the echo client and server sample app working on the TI launchpad but I cannot seem to find where to change the settings for this network like beacon mode, network speed, channels.
Can anyone point me in the right direction?

thanks,
Marijn

**Comment by jukkar** [comment]
The 802.15.4 parameters can be changed using net_mgmt API. See `include/net/ieee802154_mgmt.h` for details.

**Comment by marijnacht** [comment]
Hi Jukkar, Thanks for the reply. I don't see things there like network mode (beacon/non-beacon, or Frequency hopping). Or low power mode (RX off when idle).

**Reply by marijnacht** [reply]
Is there any support in the 802.15.4 stack for Beacon mode or has this not (yet) been implemented? Same questions actually also for TSCH and 6lowpan.
